src/components/helpers/hooks/PageNotWorkingURLHook.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from 'react';
+import { getSdkConfig } from '@pega/react-sdk-components/lib/components/helpers/config_access';
+
+export default function usePageNotWorkingURL(){
```

Comment:
rename this to hmrcExternalLinks

Reply:
Also use this hook in src\samples\ChildBenefitsClaim\AccessibilityPage\index.tsx

Reply:
Done